public/style.css

```
@@ -0,0 +1,142 @@
+body {
+  font-family: "Helvetica Neue", Arial, "Hiragino Kaku Gothic ProN",
+    "Hiragino Sans", Meiryo, sans-serif;
+  font-size: 100%;
+}
+
+main {
+  margin: 0 auto;
+  max-width: 1000px;
+  & p{
+    margin: 1em 0;
+  }
+
+  & article {
+    border-radius: 0.6rem;
+    box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.1);
+    height: 100lvh;
+    margin: 0 auto;
+    padding: 30px;
+    max-width: 1000px;
+
+    & a.button {
+      align-items: center;
+      background-color: #efefef;
+      border: 1px solid #767676;
+      border-radius: 3px;
+      display: flex;
+      height: 3em;
+      justify-content: center;      
+      text-decoration: none;
+      width: 8em;
+      &:hover{
+        background: #dbdbdb;
+      }
+      &:visited{
+        color: #000;
+      }
+    }
+
+    @media (min-width: 400px) {
+      width: 96%;
+    }
+    & header {
+      align-items: center;
+      border-bottom: 1px solid #cdcdcd;
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: space-between;
+      margin-bottom: 1rem;
+      width: 100%;
+      padding-bottom: 1.2rem;
+      & h1{
+        color: red;
+        & a {
+          text-decoration: none;
+        }
+      }
+      & p {
+        margin: 0;
+      }
+    }
+    & ul {
+      list-style-type: disc;
+      & li {
+        margin: 0.2em 0;
+        & a {
+          padding: 2px 6px;
+        }
+        & a:hover {
+          background: #eee;
+          border-radius: 4px;
+          transition: all 0.2s;
+          padding: 2px 6px;
+        }
+      }
+    }
+    & div#form,
```

```suggestion
    & #form,
```

idなのでdivは不要ですね。

それとは別に、styleを当てる場合はidよりもclassを使った方がいいかもしれません。